wgsl/index.bs

```diff
@@ -306,10 +306,11 @@ This is a portability hazard, as different implementations may exhibit the diffe
 The design of WGSL aims to minimize such cases, but is constrained by feasibility,
 and goals for achieving high performance across a broad range of devices.
 
-WGSL sometimes imposes a <dfn noexport>behavioral requirement</dfn> on
-implementations.
-These are behaviors an implementation must satisfy in order to be conformant
-with the specification.
+<dfn noexport>Behavioral requirements</dfn> are actions the implementation will
+perform when processing or executing a WGSL program. They describe the
+implementation's obligations in the contract with the programmer.
+A subset of these obligations are explicitly called out in the specifcation
```

Comment:
Typo in "specification" on line 312.

Also, suggest this rephrasing to shorten lines 312 and 313.

> The specification explicitly states these obligations when they might not be otherwise obvious.